### PR TITLE
Opacity

### PR DESCRIPTION
**Pull request by stdavis**
@steveoh When I tried to rebase this against master I was getting conflicts on every commit (33). Is that the only way to get this in? Any other ideas.

### CODE REVIEW

**Comment by steveoh** [comment]
i'll review this in a bit. i'm curious about all of the history with this pr.

**Comment by stdavis** [comment]
OK, @steveoh this is ready for a code review if you've got a minute. I just merged master rather than digging through all of the rebase conflicts. Not sure if that was the right thing to do or not. Hoping that we can rebase this after it's merged back into master or something?

File: src/app/project/FeaturesGrid.js [hunk 1]
```diff
@@ -69,12 +74,14 @@ define([
                     renderExpando: true
                 },
                 action: 'Action',
-                subType: 'Treatment/Type'
+                subType: 'Treatment/Type',
+                opacity: {
+                    label: 'Opacity',
```
Comment (steveoh) [inline src/app/project/FeaturesGrid.js]: I would like to see this column have no header and only take up as much width as the child widget. which would be around 30px for https://github.com/agrc/wri-web/blob/opacity/src/app/project/resources/FeaturesGrid.styl#L16

Comment (stdavis) [inline src/app/project/FeaturesGrid.js]: Something like this?
image

Comment (stdavis) [inline src/app/project/FeaturesGrid.js]: Or without the header cell at all?
image

**Pull request merged**